jscomp/others/belt_Array.mli

```diff
@@ -2,11 +2,11 @@
 (*                                                                     *)
 (*                                OCaml                                *)
 (*                                                                     *)
-(*            Xavier Leroy, projet Cristal, INRIA Rocquencourt         *)
+(*            Xavier Leroy;projet Cristal;INRIA Rocquencourt         *)
 (*                                                                     *)
 (*  Copyright 1996 Institut National de Recherche en Informatique et   *)
 (*  en Automatique.  All rights reserved.  This file is distributed    *)
-(*  under the terms of the GNU Library General Public License, with    *)
+(*  under the terms of the GNU Library General Public License;with    *)
 (*  the special exception on linking described in file ../LICENSE.     *)
```

Comment:
same

Reply:
Strangely, I do not recall having edited those lines. Do you need me to edit them back as they were?